flcore/datasets.py

```diff
@@ -400,7 +401,11 @@ def preprocess_data(data, column_transformer):
     n_males = len(X_train[X_train['Sex'] == 1])
     print(f'Center {center_id} of size {len(X_train)} with n_females {n_females} and n_males {n_males} in training set')
     # xx
-
+    #print("DATASETS::KAGLE::PRINT SHAPES")
+    #print(X_train.shape, y_train.shape, X_test.shape, y_test.shape)
+    #(734, 9) (734,) (184, 9) (184,)
+    #print(type(X_train), type(y_train), type(X_test),type(y_test))
+    #<class 'pandas.core.frame.DataFrame'> <class 'pandas.core.series.Series'>
```

Comment:
Please try to avoid adding changes that are not connected to the functions you contribute.